neps/nep-0171.md

```diff
@@ -276,6 +276,15 @@ interface NftTransferLog {
     memo?: string
 }
 
+// An event log to capture token metadata updating
```

Comment:
```suggestion
// An event emitted when token metadata is updated. 
```

Reply:
I don't think _log_ is appropriate here. This is an object, not a log.

Reply:
I don't think it's a big deal, probably easier to just keep it consistent with existing comments. I personally don't find it confusing.

Reply:
logging and event != emitting an event. In blockchian context we talk about emitting an event usually. So if we want to be consistent with that, then we should use _emit_

Reply:
@robert-zaremba Let's create a separate PR with this improvement to the terminology, so we can update all the Events NEPs at once and discuss it separately from this NEP.